### ISSUE TITLE
Sessions are not purged when user is deleted upon request

### ISSUE THREAD

**Issue by Hackwar** [joomla-projects/privacy-framework#161]
From this comment by @jeckodevelopment https://github.com/joomla/joomla-cms/pull/20800#issuecomment-402983550

#### Scenario

1. Submit a request of deletion (while logged in as registered user),
2. Confirm the deletion request via email
3. As admin in the backend, process the request and delete personal data.
4. User account is anonimized and data deleted as expected.
5. User account is blocked.

#### Expected result
Any active session of the user is purged and user can't login anymore.
#### Actual result
If the user has an active session, it will be anonymized: username changed to "ID XXX":
User is still logged in and can operate in the website, including visiting his profile:
After logout and trying to login again, user can't login (as expected).

**Comment by mbabker** [comment]
This one's fixed via 50c526fdd410c6c98a9700e899ec6129d9eaee0b (though admittedly I didn't actually test this one and relied 100% on writing and reading over the code several times, so someone may want to double check I didn't slip up on something stupid).

**Comment by alikon** [comment]
we have a syntax error in the SQL (missed the closing ')' )
```sql
DELETE 
FROM jlog_session
WHERE session_id IN ('i1lod9tvgc7rb762hkj904j4oj'
```

fixed by #180

**Comment by Hackwar** [comment]
In that case we should be able to just simply close this one. :smile:
